pkgs/applications/video/dvdbackup/default.nix

```diff
@@ -11,6 +11,12 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libdvdread libdvdcss dvdauthor ];
 
+  patchFlags = [ "-p0" ];
```

Comment:
`fetchpatch` itself can actually do the equivalent of this with its `stripLen` parameter, and the result allows you to use different values for different patches. It's not important, just letting you know.

Reply:
Thanks for the hint, I was missing this. However, in the `-p0` case, we still need `patchFlags` (but could use `stripLen` if another patch needs `-p1`).